examples/compiled/errorband_2d_horizontal_color_encoding.svg

```diff
@@ -1 +1 @@
ize: 10px; fill: #000; opacity: 1;">25</text><text text-anchor="end" transform="translate(-7,69.66666666666667)" style="font-family: sans-serif; font-size: 10px; fill: #000; opacity: 1;">30</text><text text-anchor="end" transform="translate(-7,28)" style="font-family: sans-serif; font-size: 10px; fill: #000; opacity: 1;">35</text></g><g class="mark-rule role-axis-domain" style="pointer-events: none;"><line transform="translate(0,200)" x2="0" y2="-200" style="fill: none; stroke: #888; stroke-width: 1; opacity: 1;"></line></g><g class="mark-text role-axis-title" style="pointer-events: none;"><text text-anchor="middle" transform="translate(-23,100) rotate(-90) translate(0,-2)" style="font-family: sans-serif; font-size: 11px; font-weight: bold; fill: #000; opacity: 1;">Miles per Gallon (95% CIs)</text></g></g><path class="foreground" d="" style="pointer-events: none; display: none; fill: none;"></path></g></g><g class="mark-area role-mark layer_0_layer_0_marks"><path d="M0,185.3448275862069L16.65525895505362,159.82142857142856L33.31051791010724,176.19047619047618L49.96577686516085,185.625L66.66666666666666,145.67901234567898L83.32192562172028,161.94444444444446L99.97718457677391,152.5765931372549L116.6324435318275,141.51785714285717L133.33333333333331,134.38078703703704L149.98859228838694,128.9676724137931L166.64385124344057,57.444683908046L200,69.63993055555548L200,46.19236111111113L166.64385124344057,15.718390804597782L149.98859228838694,87.16738505747126L133.33333333333331,96.7581018518518L116.6324435318275,103.27380952380953L99.97718457677391,121.66666666666669L83.32192562172028,133.6041666666667L66.66666666666666,108.02469135802468L49.96577686516085,162.91666666666666L33.31051791010724,143.45238095238096L16.65525895505362,119.33779761904763L0,153.44827586206898Z" style="fill: black; opacity: 0.3;"></path></g><g class="mark-line role-mark layer_0_layer_1_marks"><path d="M0,185.3448275862069L16.65525895505362,159.82142857142856L33.31051791010724,176.19047619047618L49.96577686516085,185.625L66.66666666666666,145.67901234567898L83.32192562172028,161.94444444444446L99.97718457677391,152.5765931372549L116.6324435318275,141.51785714285717L133.33333333333331,134.38078703703704L149.98859228838694,128.9676724137931L166.64385124344057,57.444683908046L200,69.63993055555548" style="fill: none; stroke: black; stroke-width: 2;"></path></g><g class="mark-line role-mark layer_0_layer_2_marks"><path d="M0,153.44827586206898L16.65525895505362,119.33779761904763L33.31051791010724,143.45238095238096L49.96577686516085,162.91666666666666L66.66666666666666,108.02469135802468L83.32192562172028,133.6041666666667L99.97718457677391,121.66666666666669L116.6324435318275,103.27380952380953L133.33333333333331,96.7581018518518L149.98859228838694,87.16738505747126L166.64385124344057,15.718390804597782L200,46.19236111111113" style="fill: none; stroke: black; stroke-width: 2;"></path></g></g><path class="foreground" d="" style="display: none; fill: none;"></path></g></g></g></svg>
```

Comment:
@kanitw @domoritz any idea why this would change so drastically without the vega spec changing?

Reply:
*merge issue